### ISSUE TITLE
[a11y] Cannot view multiple COs in Collecting Information

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1468]
[Image here](https://drive.google.com/open?id=1YOP2XP9POa1Qwh6lq9iP_LBsgFmzSG97&authuser=grant%40specifysoftware.org&usp=drive_fs)

In `a11y` there is not a way to navigate between Collection Object records on the Collecting Information form.

db: fwri
username: plarson

Windows 10, Edge

**Comment by grantfitzsimmons** [comment]
![image](https://user-images.githubusercontent.com/37256050/167497798-54980986-ea8d-4bc9-af6c-3316e275da41.png)

There needs to be a way to navigate these items as well
